data/json/itemgroups/guns.json

```diff
@@ -159,6 +159,7 @@
       [ "fn_fal", 40],
       [ "hk_g3", 40 ],
       [ "hk_g36", 30 ],
+      [ "henry_big_boy", 30],
```

Comment:
It shouldn't be as common as G3 and G36. I don't want to rebalance all of guns right now so it can go through as 10.